bidwire/notifiers/boston_bid_notifier.py

```diff
@@ -1,4 +1,4 @@
-from .notifier_utils import make_bid_item_body
```

Comment:
Please revert all changes to this file, since it is not relevant to your PR. If you'd like to propose refactoring of existing code, it is best to do that in a separate PR.